src/tq.rs

```diff
@@ -68,7 +68,7 @@ where
                             .map(|x| x / ratio.denominator)
                     }) {
                         None => MAX,
-                        Some(x) => cmp::min(MAX, x),
+                        Some(x) => cmp::min(MAX, x + 1),
```

Comment:
Could you add a comment in code explaining why we don't want to use the value `0`? Citing the reference manual.

Also I think it would be better to min-max clamp this instead of modifying the computed value, as in use `cmp::min(MAX, cmp::max(x, 1))`.

Reply:
Clamping with `min` was my initial idea, but I realized that this way the interrupt is scheduled one tick too early, see example with `instant - now == 99` and ratio 1/25 in the fourth paragraph.

Unless I misunderstand something, in this situation the reload value would be set to 3 and the handler would run 75 `Instant` ticks later, still 24 ticks too early. In the next call, `instant > now`, `(instant - now) / ratio.denominator` is 0 and we reschedule again.

Reply:
I just noticed an off-by-one in my own calculations above: after the `reset_current()`, there's one more tick from 0 to reload value that I didn't include.

Confirmed that you're right by adding a global counter increased from `dequeue` and printed after ~10 seconds. The count with `max(x, 1)` is a _little_ bit smaller than counter with `x+1`, but the difference is too small to be caused by the effect I was suggesting.